src/community/mbstyle/src/test/java/org/geoserver/community/mbstyle/web/MBStyleEditPageTest.java

```diff
@@ -0,0 +1,90 @@
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.community.mbstyle.web;
+
+import static org.junit.Assert.*;
+
+import org.apache.wicket.util.tester.FormTester;
+import org.geoserver.catalog.*;
+
+import org.geoserver.catalog.impl.StyleInfoImpl;
+import org.geoserver.community.mbstyle.MBStyleHandler;
+import org.geoserver.web.GeoServerWicketTestSupport;
+import org.junit.Before;
+import org.junit.Test;
+import org.geoserver.wms.web.data.StyleEditPage;
+import org.geotools.styling.ExternalGraphic;
+import org.geotools.styling.PointSymbolizer;
+import org.geoserver.wms.web.data.OpenLayersPreviewPanel;
+
+
+public class MBStyleEditPageTest extends GeoServerWicketTestSupport {
+    
+    StyleInfo mbstyle;
+    StyleEditPage edit;
+
+    @Before
+    public void setUp() throws Exception {
+        Catalog catalog = getCatalog();
+    	    login();
```

Comment:
I still see tabs here - you might want to take a look at your IDE settings and verify tabs to spaces and visible whitespace are turned on.

Reply:
Indeed, I haven't adjusted my settings since upgrading to the new version of Eclipse. Took some time to set up my preferences properly.